x-pack/plugins/ml/public/application/model_management/deployment_setup.tsx

```diff
@@ -63,300 +129,493 @@ export const DeploymentSetup: FC<DeploymentSetupProps> = ({
+        }
+      } else {
+        // Without adaptive resources
+        switch (config.vCPUUsage) {
+          case 'low':
+            return i18n.translate(
+              'xpack.ml.trainedModels.modelsList.startDeployment.hardwareLimits.lowCpuStaticHelp',
+              {
+                defaultMessage:
+                  'This level limits resources to the minimum required for ELSER to run. It may not be sufficient for a production application.',
+              }
+            );
+          case 'medium':
+            return i18n.translate(
+              'xpack.ml.trainedModels.modelsList.startDeployment.hardwareLimits.mediumCpuStaticHelp',
+              {
+                defaultMessage: 'Your model will consume {staticVCPUs} vCPUs consistently.',
+                values: { staticVCPUs: vcpuRange.static },
+              }
+            );
+          case 'high':
+            return i18n.translate(
+              'xpack.ml.trainedModels.modelsList.startDeployment.hardwareLimits.highCpuStaticHelp',
+              {
+                defaultMessage:
+                  'Your model will consume {staticVCPUs} vCPUs - the maximum available number.',
+                values: { staticVCPUs: vcpuRange.static },
+              }
+            );
+        }
+      }
+    } else if (!showNodeInfo) {
+      // Running a Search project in serverless
+      if (config.adaptiveResources) {
+        // With adaptive resources
+        switch (config.vCPUUsage) {
+          case 'low':
+            return i18n.translate(
+              'xpack.ml.trainedModels.modelsList.startDeployment.serverless.lowCpuAdaptiveHelp',
+              {
+                defaultMessage:
+                  'This level limits resources to {vcus, plural, one {VCU} other {VCUs}}, which may be suitable for development, testing, and demos depending on your parameters. It is not recommended for production use.',
+                values: { vcus: vcpuRange.max },
+              }
+            );
+          case 'medium':
+            return i18n.translate(
+              'xpack.ml.trainedModels.modelsList.startDeployment.serverless.mediumCpuAdaptiveHelp',
+              {
+                defaultMessage:
+                  'Your model will scale up to a maximum of {vcus, plural, one {VCU} other {VCUs}} per hour if required by your search or ingest load. Note that it will scale down automatically as needed and that you only pay for what you use.',
+                values: { vcus: vcpuRange.max },
+              }
+            );
+          case 'high':
+            return i18n.translate(
+              'xpack.ml.trainedModels.modelsList.startDeployment.serverless.highCpuAdaptiveHelp',
+              {
+                defaultMessage:
+                  'Your model will scale up to a maximum of {vcus, plural, one {VCU} other {VCUs}} per hour if required by your search or ingest load. Note that it will scale down automatically as needed and that you only pay for what you use.',
```

Comment:
```suggestion
                  'Your model will scale up to a maximum of {vcus, plural, one {VCU} other {VCUs}} per hour based on your search or ingest load. It will automatically scale down when demand decreases, and you only pay for the resources you use.',
```

Reply:
Updated in c3077e11b02d01f3be0acf409a86f4fc0cbb4a43